<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PSVRControl</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/psvr.ico</normaloff>res/psvr.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbxFunctions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Functions</string>
    </property>
    <widget class="QPushButton" name="pbtHeadsetOn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Headset On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtHeadsetOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Headset Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtEnableVRTracking">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>VR with Tracking</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtEnableVR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>VR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtCinematic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cinematic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtRecenter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Recenter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtShutdown">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Shutdown</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxStatus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="lblSerial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Device Serial : unknown</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFirmware">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Firmware Version : unknown</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbHeadsetOn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Headset On</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbHeadsetWorn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Headset Worn</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbCinematic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cinematic </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbHeadphones">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Headphones</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbMute">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mute Mic</string>
     </property>
    </widget>
    <widget class="QSlider" name="hslVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxCinematic">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>101</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Cinematic</string>
    </property>
    <widget class="QSlider" name="vslSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>22</width>
       <height>141</height>
      </rect>
     </property>
     <property name="minimum">
      <number>26</number>
     </property>
     <property name="maximum">
      <number>80</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslDistance">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>49</y>
       <width>22</width>
       <height>141</height>
      </rect>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslBrightness">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>49</y>
       <width>22</width>
       <height>141</height>
      </rect>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/icoSize.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblDistance">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/icoDistance.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblBrightness">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/icoBrigtness.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxLeds">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>LED's</string>
    </property>
    <widget class="QLabel" name="lblBrightness_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/LEDIdentification.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED A</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED B</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedC">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED C</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedD">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED D</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedE">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED E</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedF">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED F</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedG">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED G</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedH">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED H</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="vslLedH_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>22</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>LED I</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbtApply">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
